cmd/processor/processor.go

```diff
@@ -0,0 +1,122 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"github.com/supressionstop/xenking_test_1/internal/config"
+	internalLogger "github.com/supressionstop/xenking_test_1/internal/logger"
+	"github.com/supressionstop/xenking_test_1/internal/provider"
+	"github.com/supressionstop/xenking_test_1/internal/server"
+	"github.com/supressionstop/xenking_test_1/internal/storage"
+	"github.com/supressionstop/xenking_test_1/internal/usecase"
+	"github.com/supressionstop/xenking_test_1/internal/usecase/repository"
+	"github.com/supressionstop/xenking_test_1/internal/worker"
+	"log"
+	"log/slog"
+	"net/http"
+	"os"
+	"os/signal"
+	"syscall"
+)
+
+func main() {
+	ctx := context.Background()
```

Comment:
TODO: remove run(), move behaviour in main().

My reason to use run() is in returning error, instead of exiting app.
The idea of run() function came out of this article:
https://grafana.com/blog/2024/02/09/how-i-write-http-services-in-go-after-13-years/#func-main-only-calls-run